backend/mora/service/addresses/dar.py

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import requests
+import flask
+from . import base
+from ... import mapping
+
+session = requests.Session()
+
+NOT_FOUND = "Ukendt"
+
+
+class DARAddressHandler(base.AddressHandler):
+    scope = 'DAR'
+    prefix = 'urn:dar:'
+
+    def __init__(self, value):
+        super().__init__(value)
+
+        try:
+            self.address_object = self._fetch_from_dar(value)
+        except Exception as exc:
+            self.address_object = {
```

Comment:
We can do better than this, now!

Previously, the only useful data we could return when offline, was some text explaining that we couldn't, in fact, return anything useful. Now, we have an entire object where we can stash all sorts of useful data, such as a stringification off the actual address. If we do so, we might even be able to avoid the step of looking up in DAR when reading, although that's not of great importance. But we could simplify the reading code: the only thing DAWA's `historik` endpoints offer us is a textual description, but if we already have that, why query it?

Reply:
I feel like these improvements, while seeming like very good ideas, are kind of out of scope of this feature. As of now, I've done my best to reimplement the existing functionality, which appears to be working. I suggest we implement these ideas at a later date.